pandas/core/common.py

```diff
@@ -155,7 +155,7 @@ def cast_scalar_indexer(val):
     outval : scalar
     """
     # assumes lib.is_scalar(val)
-    if lib.is_float(val) and val.is_integer():
+    if (lib.is_float(val) and val.is_integer()) or lib.is_integer(val):
```

Comment:
i guess this is to convert np.int64 to int?  Can you comment on why this is needed?

Reply:
This is no longer needed, but there are certain tests ensuring we're returning `int` rather than `np.int64`